### PR TITLE
Add cases to check secret-set-value --file and --interactive 

### PR DESCRIPTION
**Pull request by Yingshun**
This PR adds 3 cases:
1) Set a secret value with --file option.
2) Set a secret value with --file and --plain option.
3) Set a secret value with --interactive option.

depends on 
https://github.com/avocado-framework/avocado-vt/pull/2832
https://github.com/avocado-framework/avocado-vt/pull/2833

Signed-off-by: Yingshun Cui <yicui@redhat.com>

### CODE REVIEW

**Comment by Yingshun** [comment]
```
# avocado run --vt-type libvirt --vt-machine-type q35 virsh.secret_set --vt-connect-uri qemu:///system
JOB ID     : 9e8ad024fabb3119b6d98b3b9ec3b314081898f3
JOB LOG    : /root/avocado/job-results/job-2020-11-22T20.15-9e8ad02/job.log

 (1/1) type_specific.io-github-autotest-libvirt.virsh.secret_set.positive_testing.set_secret_value.file.public_secret: PASS (3.73 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 5.05 s

```

**Comment by chunfuwen** [comment]
If the case is to just validate --file option, I assume '--file' option can be implicitly covered by using virsh.secret_set_value in other cfg file, such as  https://github.com/autotest/tp-libvirt/pull/3134/files

**Comment by Yingshun** [comment]
> If the case is to just validate --file option, I assume '--file' option can be implicitly covered by using virsh.secret_set_value in other cfg file, such as https://github.com/autotest/tp-libvirt/pull/3134/files

Thanks @chunfuwen for the reminder. I've checked with feature owner, she prefers to have a test for --file option in secret job so I added it here.

**Comment by Yingshun** [comment]
@meinaLi Could you please help review this PR? Thanks!

File: libvirt/tests/cfg/virsh_cmd/secret/virsh_secret_set.cfg [hunk 1]
```diff
@@ -0,0 +1,39 @@
+- virsh.secret_set:
```
Comment (meinaLi) [inline libvirt/tests/cfg/virsh_cmd/secret/virsh_secret_set.cfg:1]: Why don't we directly use virsh_secret_set_get.cfg but create a new cfg file?

Comment (Yingshun) [reply libvirt/tests/cfg/virsh_cmd/secret/virsh_secret_set.cfg:1]: Thanks @meinaLi! The reason is that it's **hard** to expand positive tests.
I have to update caselink/jira for the existing positive cases, and further more, I have no idea what patterns they should use.

File: libvirt/tests/cfg/virsh_cmd/secret/virsh_secret_set_get.cfg [hunk 2]
```diff
@@ -35,4 +35,28 @@
...
+            status_error = "no"
+            secret_base64_no_encoded = "my_secret"
+            variants:
+                - set_secret_value:
+                    only public_secret
+                    variants:
+                        - file:
+                            secret_file = "yes"
+                            main_vm = ""
+                            vms = ""
```
[the remark this answers is not in the document]
Comment (Yingshun) [reply libvirt/tests/cfg/virsh_cmd/secret/virsh_secret_set_get.cfg:51]: The reason is that the existing cases do not need to manage the VM but 2 of new added cases do. The value of the main_vm is set in base.cfg which can be easily managed by ci, so I don't want to copy it into subtest.cfg, I prefer to keep the original style.

**Review by meinaLi — APPROVED** [review]

**Review by chloerh — APPROVED** [review]

**Pull request merged**